server/src/main/java/com/cloud/deploy/DeploymentPlanningManagerImpl.java

```diff
@@ -489,8 +487,6 @@ public DeployDestination planDeployment(VirtualMachineProfile vmProfile, Deploym
                         } else {
                             s_logger.debug("The last host of this VM does not have enough capacity");
                         }
-                    } else {
```

Comment:
Any reason to delete this log? Better to display if either tags or the guest os preference mismatches right?

Reply:
@ravening it is moved to method checkVmProfileAndHost